model/marine/marine_background_error_hybrid_diffusion_diffusion.yaml.j2

```diff
@@ -57,41 +58,44 @@ components:
     value: 1.00
 
 - covariance:
-    covariance model: ensemble
-    members from template:
-      template:
-        read_from_file: 1
-        date: '{{marine_stddev_time}}'
-        basename: ../ensdata/
-        ocn_filename: 'ocn.pert.steric.%mem%.nc'
-        ice_filename: 'ens/ice.%mem%.nc'
-        state variables:
-        - sea_ice_area_fraction
-        - sea_ice_thickness
-        - sea_ice_snow_thickness
-        - sea_water_salinity
-        - sea_water_potential_temperature
-        #- eastward_sea_water_velocity
-        #- northward_sea_water_velocity
-        - sea_surface_height_above_geoid
-      pattern: '%mem%'
-      nmembers: '{{marine_number_ensemble_members}}'
-    localization:
-      localization method: SABER
-      saber central block:
-        saber block name: diffusion
-        read:
-          groups:
-          - variables:
-            - sea_water_potential_temperature
-            - sea_water_salinity
-            - sea_surface_height_above_geoid
-            - sea_ice_area_fraction
-            horizontal:
-              filepath: ./staticb/hz_ocean
-            vertical:
-              levels: {{marine_vt_levels}}
-              filepath: ./staticb/vt_ocean
+    covariance model: SABER
+    ensemble:
+      members from template:
+        template:
+          read_from_file: 1
+          date: '2021-07-01T00:00:00Z'
+          basename: ../ensdata/
+          ocn_filename: ocn.pert.%mem%.nc
+          ice_filename: ice.pert.%mem%.nc
+          state variables:
+          - sea_water_potential_temperature
+          - sea_water_salinity
+          #- eastward_sea_water_velocity
+          #- northward_sea_water_velocity
+          - sea_surface_height_above_geoid
+          - sea_ice_area_fraction
+          - sea_ice_thickness
+          - sea_ice_snow_thickness
+        pattern: '%mem%'
+        nmembers: '{{marine_number_ensemble_members}}'
+    saber central block:
+      saber block name: Ensemble
+      localization:
+        saber central block:
+          saber block name: diffusion
+          read:
+            groups:
+            - variables:
+              - sea_water_potential_temperature
+              - sea_water_salinity
+              - sea_surface_height_above_geoid
+              - sea_ice_area_fraction
+              multivariate strategy: duplicated
+              horizontal:
+                filepath: ./staticb/hz_ocean
+              vertical:
+                strategy: duplicated
```

Comment:
vertical localization changed here compared to previous configuration, is that intentional?

Reply:
yes, we don't want vertical localization.